src/Cassandra/Orleans.Clustering.Cassandra/Hosting/CassandraClusteringOptions.cs

```diff
@@ -10,6 +12,20 @@ namespace Orleans.Clustering.Cassandra.Hosting;
 /// </summary>
 public class CassandraClusteringOptions
 {
+    /// <summary>
+    /// Optionally configure time-to-live behavior for the membership table row data in Cassandra itself, allowing
+    /// defunct silo cleanup even if a cluster is no longer running. The time-to-live time will be calculated as a
+    /// multiple of the IAmAlive <c>AllowedIAmAliveMissPeriod</c>.
+    /// </summary>
+    /// <remarks>
```

Comment:
Include comment on >=10 and explanation that it is somewhat arbitrary but is intended to be well larger than the IAmAlive failure time.

And actually, do we need to expose this as configuration at all? In thinking of what documentation would include to guide users on picking a value here, I can't really think of situations when it would be advantageous to lower or raise this value. Maybe there can just be a bool (or enum?) that's exposed to turn this on or off.

Reply:
I would be ok with changing this to a boolean on/off instead, and having the "on" TTL value be 10x `AllowedIAmAliveMissPeriod`